test_util/launch.py

```diff
@@ -1,20 +1,282 @@
-"""
+"""DC/OS Launch
 
 Usage:
-dcos-launch create [--wait] [--dump-info=cluster_info.json] <config=config.yaml>
-dcos-launch wait <path=cluster_info.json>
-dcos-launch describe <path=cluster_info.json>
-dcos-launch delete <path=cluster_info.json>
+  dcos-launch create [--config-path=<path>] [--info-path=<path>]
+  dcos-launch wait [--info-path=<path>]
+  dcos-launch describe [--info-path=<path>]
+  dcos-launch test [--info-path=<path>] [--] [<test_command>]...
+  dcos-launch delete [--info-path=<path>]
+
+Commands:
+  create    reads the file given by --config-path, creates the cluster described therein
+            and finally dumps a JSON file to the path given in --info-path which can then
+            be used with the wait, describe, and delete calls.
+  wait      Block until the cluster is up and running.
+  describe  Return additional information about the composition of the cluster.
+  delete    Destroying the provided cluster deployment.
+  test      Runs integration test suite on cluster. Can optionally supply a specific test command.
+
+Options:
+  --config-path=<path>  Path for config to create cluster from [default: config.yaml].
+  --info-path=<path>    JSON file output by create and consumed by wait, describe,
+                        and delete [default: cluster_info.json].
 """
+import abc
+import copy
+import os
+import sys
 
 from docopt import docopt
 
+import ssh.tunnel
+import test_util.runner
+from pkgpanda.util import json_prettyprint, load_json, load_string, load_yaml, write_json
+from test_util.aws import BotoWrapper, DcosCfSimple
+
+DCOS_LAUNCH_VERSION = '0.1'
```

Comment:
Why does this have independent versioning / need a versioning streem of its own? Isn't doing any sort of independent release process from the core DC/OS release, and as far as I'm aware there's no plan to do that. I'd drop the versioning string altoghether.